### ISSUE TITLE
Modals are not hidden from DOM even when they are not opened

### ISSUE THREAD

**Issue by hannyle** [CSCfi/swift-browser-ui#887]
#### Describe the bug

Using keyboard to navigate through SD Connect's browser page, we can tab on elements/components inside the modals which are actually not visible and should not be available for keyboard navigation.

#### To Reproduce
Steps to reproduce the behaviour:
1. Go to SD Connect's browser page
2. Use keyboard to tab
3. Recognise that some invisible elements can be interacted with the Tab key

#### Expected behavior

Modals that are not opened should be totally hidden from DOM and Tab key cannot interact with them.
